source/2019-11-29-the-ember-times-issue-126.md

```diff
@@ -45,13 +47,11 @@ All questions are optional and reaching 100% completion is not required so give
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [All Things Ember and Tailwind 🎨](https://www.balinterdi.com/blog/purging-css-in-ember-postcss-purgecss/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Are you on the functional CSS train? 🚂 Tailwind CSS is a highly customizable, low-level CSS framework that gives you all of the building blocks you need to build bespoke designs without any annoying opinionated styles you have to fight to override. [Balint Erdi (@balinterdi)](https://github.com/balinterdi) recently blogged about Ember, Tailwind and PostCSS. In [his first post](https://www.balinterdi.com/blog/ember-tailwind-css-postcss-import/), Balint explains how with [ember-auto-import](https://github.com/ef4/ember-auto-import), you can use [Tailwind](https://tailwindcss.com/) directly versus relying on the Ember-specific integrator package. (ember-cli-tailwind was deprecated for this reason!) However, to make your Ember app integrate nicely with Post CSS, Baling suggests sticking with the integrator add-on [ember-cli-postcss](https://github.com/jeffjewiss/ember-cli-postcss). Check out [post-css-import](https://github.com/postcss/postcss-import) to be able to import styles from other files.
```

Comment:
- I would move the 🚂 emoji in front of the `?` 
- You misspelled Balint his name in the last line.
- Should the sentence between `( ember-cli-tailwind)` be after the `.` instead of before? And at least for me as a novice reader I don't understand this. Maybe split the sentence into In his blogpost he showed how to do it with Ember auto import. Next sentence The intergrator package was deprecated because this worked so well.

Reply:
thank you!! 

i _think_ how i changed it matches what you said, but let me know what you think. i'm going to merge it in since on holiday, but can def make other changes on the main branch!